pertpy/tools/_perturbation_space/_discriminator_classifiers.py

```diff
@@ -149,32 +148,48 @@ def load(  # type: ignore
         batch_size: int = 256,
         test_split_size: float = 0.2,
         validation_split_size: float = 0.25,
-    ):
-        """Creates a classifier model and dataloaders required for training and testing.
+        max_epochs: int = 40,
+        val_epochs_check: int = 5,
```

Comment:
Do normal users actually care about a validation check?

Reply:
It's only important for early stopping, as that is influenced by both the `val_epochs_check` and the `patience` parameter. Training will stop after `patience` validation checks without improvement. If validation is only checked every 20 epochs, then implementing early stopping is basically pointless. However, I'll reduce the default here because also a value of 5 is quite high based on what we've seen for the datasets so far.

Reply:
Ahh I see. Fair!